helper-cli/src/main/kotlin/utils/PathExcludeGenerator.kt

```diff
@@ -99,7 +100,7 @@ internal object PathExcludeGenerator {
             ?: return emptySet()
 
         return createExcludePattern(
-            directory = getClosestCommonAncestor(matchingFiles).path,
+            directory = getCommonParentFile(matchingFiles).path,
```

Comment:
This breaks one of the unit test in PathExcludeGeneratorTest (at least on Linux).